util/auth.go

```diff
@@ -0,0 +1,10 @@
+package util
+
+import (
+	"encoding/base64"
+	"fmt"
+)
+
+func BasicAuth(username, password string) string {
```

Comment:
1- Use more semantic name, like: `MakeBasicCredential`
Read more here: https://datatracker.ietf.org/doc/html/rfc7617

2- Mode it to `basicauth` package. Read above